### PR TITLE
[DA-31] Add logger dependencies

### PR DESCRIPTION
**Pull request by jomrazek**

### CODE REVIEW

File: reports-rest/pom.xml [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+		<artifactId>parent</artifactId>
+		<version>0.0.1-SNAPSHOT</version>
+	</parent>
+	<artifactId>reports-rest</artifactId>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.jboss.logging</groupId>
+			<artifactId>jboss-logging</artifactId>
+		</dependency>
```
Comment (rnc) [inline reports-rest/pom.xml]: PNC has just standardised on SLF4J (See NCL-908 and https://github.com/project-ncl/pnc/pull/394 ) : perhaps the same should be done here? 

Why the large whitespace change?

Comment (jomrazek) [inline reports-rest/pom.xml]: Ok, I can look at it and we can change it, whitespace change is caused by formating i don't know how Jakub format it. When i edit what I need I still have changes so i reformat all xmls cause they were inconsistent anyway.

Comment (jbartece) [inline reports-rest/pom.xml]: I had some weird format setup

**Comment by jbartece** [comment]
Rebase to master please. There are significant changes in the POM files.

**Pull request closed without merging**